src/artkit/model/llm/vertexai/_vertexai.py

```diff
@@ -0,0 +1,213 @@
+# -----------------------------------------------------------------------------
+# © 2024 Boston Consulting Group. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+
+"""
+VertexAI LLM systems.
+"""
+from __future__ import annotations
+
+import logging
+from abc import ABCMeta
+from collections.abc import Mapping
+from contextlib import AsyncExitStack
+from typing import Any, TypeVar
+
+from artkit.model.llm.history._history import ChatHistory
+from pytools.api import MissingClassMeta, appenddoc, inheritdoc, subsdoc
+
+from ...util import RateLimitException
+from ..base import ChatModelConnector
+
+logger = logging.getLogger(__name__)
+
+__all__ = ["VertexAIChat"]
+
+try:
+    # note that vertexai is imported through the google-cloud-aiplatform package
```

Comment:
Let's update comment to explicitly say that pip installing google-cloud-aiplatform includes vertexai and that we recommend this instead of installing vertexai individually because the package is not available in conda and our repo utilizes conda builds/tests